resources/views/dashboard/controllers/events/view.blade.php

```diff
@@ -65,96 +71,121 @@
+                                                                </div>
                                                             </div>
                                                         </div>
-                                                    </div>
-                                                    <div class="form-group">
-                                                        <div class="row">
-                                                            <div class="col-sm-6">
-                                                                {!! Form::label('start_time', 'Start Time (Zulu)') !!}
-                                                                {!! Form::text('start_time', $r->start_time, ['placeholder' => $event->start_time, 'class' => 'form-control']) !!}
+                                                        <div class="form-group">
+                                                            <div class="row">
+                                                                <div class="col-sm-12">
+                                                                    {!! Form::label('position_detail', 'Position or sector ID assigned') !!}
+                                                                    {!! Form::text('position_detail', null, ['class' => 'form-control']) !!}
+                                                                </div>
                                                             </div>
-                                                            <div class="col-sm-6">
-                                                                {!! Form::label('end_time', 'End Time (Zulu)') !!}
-                                                                {!! Form::text('end_time', $r->end_time, ['placeholder' => $event->end_time, 'class' => 'form-control']) !!}
+                                                        </div>
+                                                        <div class="form-group">
+                                                            <div class="row">
+                                                                <div class="col-sm-6">
+                                                                    {!! Form::label('start_time', 'Start Time (Zulu)') !!}
+                                                                    {!! Form::text('start_time', $r->start_time, ['placeholder' => $event->start_time, 'class' => 'form-control']) !!}
+                                                                </div>
+                                                                <div class="col-sm-6">
+                                                                    {!! Form::label('end_time', 'End Time (Zulu)') !!}
+                                                                    {!! Form::text('end_time', $r->end_time, ['placeholder' => $event->end_time, 'class' => 'form-control']) !!}
```

Comment:
The start/end time placeholder should be the start/end of the event. For some reason, this is not working properly - just showing 00:00 for both fields.

Reply:
This should be resolved in the newest commit - start/end times will be shown in either local or zulu depending on what box is selected. This is not a validation error - it is intentional that leaving both boxes blank will select you as available for the whole event, for simplicity. Keeping this unresolved for now, but will mark resolved if you think my newest change is ok